client-side/src/stories/Select/select.scss

```diff
@@ -4,7 +4,7 @@
 
   .genericSelect {
     &.primary {
-      background-color: $background-class;
+      // background-color: $background-class;
```

Comment:
why did you comment this?

Reply:
זה עשה שגיאה זה נראה שמשהי מחקה מהVarables את הצבע הזה, זה צבע לבן וזה עובד מעולה כשזה בהערה , אולי פשוט אפשר למחוק את זה?